static/bidder-info/nativo.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://exchange.postrelease.com/esi?ntv_epid=7"
+maintainer:
+  email: "prebiddev@nativo.com"
+gvlVendorID: 263
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
+    url: http://jadserve.postrelease.com/suid/101787?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&ntv_gpp_consent={{.GPP}}&ntv_r={{.RedirectURL}}
+    userMacro: "NTV_USER_ID"
```

Comment:
@rafataveira  can redirect url be `https` endpoint?

Reply:
done